mediapipe/examples/python/ovms_object_detection.py

```diff
@@ -0,0 +1,6 @@
+import mediapipe as mp
+ovms_object_detection = mp.solutions.ovms_object_detection
+with ovms_object_detection.OvmsObjectDetection(side_inputs=
+        {'input_video_path':'/mediapipe/mediapipe/examples/desktop/object_detection/test_video.mp4',
```

Comment:
empty line

Reply:
between parameters ? This would look strange.